cookbook/security/voters_data_permission.rst

```diff
@@ -104,8 +104,8 @@ edit a particular object. Here's an example implementation:
 
                     break;
                 case self::EDIT:
-                    // we assume that our data object has a method getOwner() to
-                    // get the current owner user entity for this data object
+                    // this assumes that the data object has a method getOwner()
+                    // to get the current owner user entity for this data object
```

Comment:
this is a bit difficult to read -> `the current owner user entity for this data object`